scripts/ci/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+readonly dir=$(dirname $0)
+readonly projectRoot=$dir/../..
+
+if [ -n "$GCLOUD_FILE" ]; then
+  source $dir/gcloud-init.sh
+fi
+
+if [ -n "$DOCKER_NAMESPACE" ]; then
+  projectName="$DOCKER_NAMESPACE"
```

Comment:
I'd suggest using a different variable name instead of `projectName`, since `gcr.io/PROJECT_NAME` is more than just the project.